### PR TITLE
[OREE-814] Extensibility: retire beacon feature

### PR DESCRIPTION
**Pull request by sheff3rd**
# [OREE-814](https://outreach-io.atlassian.net/browse/OREE-814)

## Description
The beacon feature (originally implemented for Terminus) should be retired as a corner case feature not used by any partner atm

[OREE-814]: https://outreach-io.atlassian.net/browse/OREE-814?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sheff3rd** [comment]
I did run `yarn pretty:fix` and it did a bunch of changes, so I decided to push them in a separate commit when possible

File: examples-be/src/main.ts [hunk 1]
```diff
@@ -56,22 +45,15 @@ class TokenService {
    * Getting installation-based access token: https://developers.outreach.io/api/app-token/#get-s2s-access-token
    * @param installationId
    */
-  public async getInstallationBasedToken(
```
Comment (honeydripper) [inline examples-be/src/main.ts:59]: @sheff3rd this file is full of formatting changes, please make sure that prettier setup is correct.

**Review by honeydripper — CHANGES_REQUESTED** [review]

File: examples-be/src/main.ts (hunk 1 above)
Comment (honeydripper) [reply examples-be/src/main.ts:59]: I think a better way of handling the prettier fixes would be a separate PR with prettier fixed and enforced as a build check. @palootcenas-outreach wdyt?

Comment (palootcenas-outreach) [reply examples-be/src/main.ts:59]: I agree

File: docs/schema/2.0/manifest.schema.json [hunk 2]
```diff
@@ -366,10 +366,6 @@
               "decoration": {
                 "type": "string",
                 "enum": ["none", "full", "simple"]
-              },
-              "notificationsUrl": {
```
[the remark this answers is not in the document]
Comment (palootcenas-outreach) [reply docs/schema/2.0/manifest.schema.json:370]: Petr confirmed.

File: src/manifest/extensions/shell/ShellExtensionHost.ts [hunk 3]
```diff
@@ -19,22 +19,6 @@ export class ShellExtensionHost extends ExtensionHost {
...
-   *
-   * @see https://github.com/getoutreach/clientxtsdk/blob/master/docs/manifest.md#notificationsUrl
-   * @type {string}
-   * @memberof ShellExtensionHost
-   */
-  notificationsUrl?: string;
-
   /**
    * Type of the decoration to be used for badge decorations.
    *  - none - no decoration badge
```
Comment (palootcenas-outreach) [inline src/manifest/extensions/shell/ShellExtensionHost.ts:24]: `decoration` below should be removed as well

File: src/legacy/ManifestTranslator.ts [hunk 4]
```diff
@@ -422,7 +421,6 @@ export class ManifestTranslator {
     }
 
     extension.host = new ShellExtensionHost();
-    extension.host.notificationsUrl = ext.host.notificationsUrl;
     extension.host.decoration = DecorationStyle.FULL;
```
Comment (palootcenas-outreach) [inline src/legacy/ManifestTranslator.ts:424]: Also remove all usages of `decoration`, includins the enum `DecorationStyle`

**Review by palootcenas-outreach — APPROVED** [review]

File: examples-be/src/main.ts (hunk 1 above)
Comment (sheff146) [reply examples-be/src/main.ts:59]: I think it's OK to merge it in the current state but for the future it's a good practice to split refactoring/formatting and functional changes into separate PRs. Makes it easier to review, easier to track changes or manage possible reverts.

**Review by sheff146 — APPROVED** [review]

**Pull request merged**